pyproject.toml

```diff
@@ -95,7 +110,10 @@ convention = "google"
 
 [tool.setuptools]
 zip-safe = false
+packages = ["fake_useragent"]
```

Comment:
is there a reason to have this named using an underscore?

Reply:
There it was needed, but probably some changes are not needed. I'm reverting some changes in pyproject. Interesting comments! :)
If you enable the CI we could see better the whole process. Locally both alternatives seem to work fine